Modules/Messages/src/Messages.Infrastructure/Persistence/Database/MessagesDbContext.cs

```diff
@@ -14,9 +15,9 @@ public class MessagesDbContext : AbstractDbContextBase
 
     public MessagesDbContext(DbContextOptions<MessagesDbContext> options, IServiceProvider serviceProvider) : base(options, serviceProvider) { }
 
-    public virtual DbSet<Message> Messages { get; set; }
-    public virtual DbSet<RecipientInformation> RecipientInformation { get; set; }
-    public virtual DbSet<Relationship> Relationships { get; set; }
+    public virtual DbSet<Message> Messages { get; set; } = null!;
+    public virtual DbSet<RecipientInformation> RecipientInformation { get; set; } = null!;
+    public virtual DbSet<Relationship> Relationships { get; set; } = null!;
```

Comment:
This class has an empty constructor (line 12) and is tied to the EF Core in a way so I think message by @tnotheis in the **Product - Backbone** team on Teams regarding the `= null!` initializers.

Reply:
Ok, I may have been wrong here. 

Timo said: *"I was ONLY referring to the DbSet properties of the DbContexts, and NOT about ANY class."*, so it may be ok to leave it here as it is.

@tnotheis am I mistaken with this?

Reply:
The current implementation is exactly how it's supposed to be. 
The properties are not set in any of the constructors. Instead, they are set via reflection on initialization of the DbContext.

Reply:
Thanks Timo for the answer. Marking this then as resolved.